app/concepts/api/v1/users/registrations/contract/validate.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module Api::V1::Users::Registrations::Contract
+  class Validate < ApplicationContract
+    params do
+      required(:email).filled(:string)
```

Comment:
What whould be for case wen email is `i_am_an_email`? I think it will pass.

Reply:
Added regex matching for email